src/ocamlorg_frontend/components/learn_overview.eml

```diff
@@ -0,0 +1,13 @@
+let skill_tag 
+difficulty =
+  match difficulty with 
+  | Some "beginner" ->
+    <div class="w-28 h-8 p-1 inline-block bg-white rounded-3xl border-2 border-teal-700 font-normal text-center text-teal-700"> Beginner  </div>
```

Comment:
The text is not centered because of the explicit height. It's better to do it with adequate padding to achieve the desired height.